qtism/runtime/expressions/OutcomeMaximumProcessor.php

```diff
@@ -94,12 +95,13 @@ public function process()
                             $result[] = new QtiFloat(floatval($normalMaximum *= $weight->getValue()));
                         }
                     } else {
-                        // If any of the items in the given subset have no declared maximum
-                        // the result is NULL.
-                        return null;
+                        // If any of the items in the given subset have no declared maximum, possibly an item require human-based scoring)
+                        // we skip to the next item
+                        continue;
```

Comment:
I'm afraid we cannot accept this part of the PR, because of the following statement about the outcomeMaximum expression in the QTI Specification:

> If any of the items within the given subset have no declared maximum the result is NULL, otherwise the result has cardinality multiple and base-type float.

I think that a strategy must be found in the assessmentTest definition we generate to circumvent this. For instance, we could tag the items that have to be taken into account (non-information) using a category "non-informational" to be used with the outcomeMaximum.

```xml
<outcomeMaximum outcomeIdentifier="SCORE" includeCategory="non-informational" weight="SOMEWEIGHT"/>
```